releases/release-1.29/release-notes/maps/pr-119396-map.yaml

```diff
@@ -1,6 +1,6 @@
 pr: 119396
 releasenote:
   text: |-
-    `kube-scheduler` implements scheduling hints for the `NodeUnschedulable` plugin.
+    `kube-scheduler` implemented scheduling hints for the `NodeUnschedulable` plugin.
```

Comment:
`kube-scheduler` will now implement scheduling hints for the `NodeUnschedulable` plugin.

Reply:
This ones also works as it is, especially without the ":" between the component and the description.